adafruit_mcp4728.py

```diff
@@ -129,17 +136,19 @@ def __init__(self, i2c_bus, address: int = MCP4728_DEFAULT_ADDRESS):
         self.channel_d = Channel(self, self._cache_page(*raw_registers[3]), 3)
 
     @staticmethod
-    def _get_flags(high_byte):
+    def _get_flags(high_byte: int) -> Tuple[int, int, int]:
         vref = (high_byte & 1 << 7) > 0
         gain = (high_byte & 1 << 4) > 0
         power_state = (high_byte & 0b011 << 5) >> 5
         return (vref, gain, power_state)
 
     @staticmethod
-    def _cache_page(value, vref, gain, power_state):
+    def _cache_page(
+        value: int, vref: int, gain: int, power_state: int
+    ) -> Dict[str, int]:
         return {"value": value, "vref": vref, "gain": gain, "power_state": power_state}
 
-    def _read_registers(self):
+    def _read_registers(self) -> Iterable[Tuple[int, int, int, int]]:
```

Comment:
`Iterable` is correct, but we could be more specific here and use `List`:

```suggestion
    def _read_registers(self) -> List[Tuple[int, int, int, int]]:
```